EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -97,6 +98,9 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usePhase1 = config_.getParameter<bool> ("UsePhase1");
     if(usePhase1) edm::LogInfo("SiPixelRawToDigi")  << " Use pilot blade data (FED 40)";
   }
+  //CablingMap could have a label //Tav
+  CablingMapLabel = ""; 
+  CablingMapLabel = config_.getParameter<std::string> ("CablingMapLabel");
```

Comment:
In the case if the CablingMapLabel is not defined in the config file then the label should be the empty one thus the line CablingMapLabel="" which is the usual case. Originally there was an if (config_.exists("CablingMapLabel")) statement which I corrected at @slava77 's request.